config/locales/forms/find_support.en.yml

```diff
@@ -0,0 +1,59 @@
+en:
+  find_support:
+    defaults:
+      options:
+        yes: "Yes"
+        soon: "I might be soon"
```

Comment:
I don't think we should have default options. I think it could get quite messy making sure options are excluded etc.

Reply:
Yeah up for a debate on this one, my thinking was many of these forms (especially things like radio boxes), re-use the same answers over and over and over. I don't want to DRY-for-DRY sake, but i figure not having these typed out each time helps keep the YAML file shorter and slimmer, which might help content folks navigate it on github?

The mental model would be:
- each question has it's own options, anything added there overrides any other locale
- anything shared can be DRYed out to shared defaults at the top. Also means we can could change all "Yes" answers to "Yep" in one place if we wanted to.

Reply:
I'm not a fan of this. I think each question should have the full amount of answers defined with the question content (like we did for the other forms). If content designers are editing the file, they often want to edit one question. Having to scroll to the top to edit "Yes" to "Yep" and not realising that that'll change for everything is potentially disastrous.

Reply:
Humm, I suppose I'm wondering if the locale file enumerates all possibilities in a non-dry way, is that then confusing when comparing a questions data configuration and a questions content?

They have to match 100% instead of just describing where you can find content for a question?

Reply:
I take the point about the potential for mix up, you still think that's a risk given the github requirement for review and the idea we could use tests to cover really unexpected outcomes?

Reply:
I'm with Issy on this. The potential for accidentally changing questions you didn't mean to is very high. We also have to account for questions that don't use any of the default options.

Reply:
Is that the checkbox question?

Reply:
I suppose the only other argument I can think of for defaults, is that if you added another radio box question, which used the same keys as other questions but forgot to add the locale file, it'd render something sensible instead of the locale error pattern.

But potentially we can use data validation tests/checks to protect against that? 🤔 

But fair if there's a strong sentiment against it, happy to change. Will hang on for another couple reviews then switch it over to the majority opinion / the consensus point?

Reply:
ok questions...
What if... i renamed the key to `fallbacks` and put it at the bottom of the file?
Is the problem that there's a risk in changes being made without realising the scope? or,
is it more about not enumerating all possibilities in the locale file?

trying to think this one through 🤔